src/index.css

```diff
@@ -66,3 +66,9 @@ button:focus-visible {
     background-color: #f9f9f9;
   }
 }
+
+/* mapcontainer */
```

Comment:
Does this map container need to be in index? Or should you create a css module for your map?

I'm asking, "Is this container css something that will be used by other devs and therefore should be open to all via index.css or is this specific to your work, and should be in it's own file?"

Reply:
Yes, it can I'll update the code.

Reply:
REMOVED from index.css